src/main.c

```diff
@@ -388,61 +88,7 @@ void main(void)
 		printk("Modem info could not be initialised: %d\n", err);
 		return;
 	}
+	nat_test_init();
```

Comment:
Nit: missing blank line before this line

Reply:
Fixed